<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Gestionmachine">
     <attribute name="title">
      <string>Gestion machine</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>581</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ajouter machine</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>601</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter machine</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Code machine</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>160</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Type machine</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Puissance</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>260</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Consommation</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="typemachine">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>160</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="codemachine">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="puissancemachine">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="consommationmachine">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>260</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEditmachine">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>110</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
       <zorder>pushButton_3</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>groupBox_3</zorder>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Afficher machine</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>601</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer machine</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>601</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer machine</string>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>180</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Code machine</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="codemachineSuppr">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>190</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>470</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>470</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Gestionreclamation">
     <attribute name="title">
      <string>Gestion reclamation</string>
     </attribute>
     <widget class="QTabWidget" name="tabR">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ajout">
       <attribute name="title">
        <string>Ajouter reclamation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>601</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter reclamation</string>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Code machine</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>130</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de panne</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>190</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Cause de panne</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="codemachinerecl">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_cause">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>190</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_panneEdit">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="addR">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="affichage">
       <attribute name="title">
        <string>Afficher reclamation</string>
       </attribute>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>601</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>480</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Modification">
       <attribute name="title">
        <string>Modifier reclamation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>601</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifer reclamation</string>
        </property>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>230</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Cause de panne</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>100</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Code machine</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="codemachinerecl_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>170</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de panne</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_cause_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>230</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_panneEdit_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>170</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="modifierREC">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="suppression">
       <attribute name="title">
        <string>Supprimer reclamation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>601</width>
          <height>391</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer reclamation</string>
        </property>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>180</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Code machine</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_code_6">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>190</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="DeleteR">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>470</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>470</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_machine">
    <property name="title">
     <string>Smart Factory</string>
    </property>
   </widget>
   <addaction name="menuGestion_machine"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
